files/build_templates/docker_image_ctl.j2

```diff
@@ -364,6 +364,9 @@ start() {
         -v /var/run/redis$DEV:/var/run/redis:rw \
         -v /var/run/redis-chassis:/var/run/redis-chassis:ro \
         -v /usr/share/sonic/device/$PLATFORM/$HWSKU/$DEV:/usr/share/sonic/hwsku:ro \
+{%- endif %}
+{%- if docker_container_name == "swss" %}
+        -v /var/log/swss$DEV:/var/log/swss:rw \
```

Comment:
this changes the folder even for single asic where we are trying to make single asic exactly the same as if there is no multi-asic setup.

Reply:
The path doesn't change for single instance containers because `$DEV` is unset in this case.